src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
```

Comment:
it s better to apply this property to all elements (use * selector)

when all elements have the same `box-sizing` value, their behavior becomes more predictable

Reply:
I fixed all the changes except this one, I can fix it, but I did it last time, which I was told was wrong.
How should I be in such a case?

https://github.com/mate-academy/layout_product-cards/pull/3328#discussion_r1269465815

Reply:
I think mentor said about margin